client/state/index.js

```diff
@@ -62,7 +63,7 @@ import wordads from './wordads/reducer';
 /**
  * Module variables
  */
-export const reducer = combineReducers( {
+const reducers = {
```

Comment:
Yep, so it could be placed below. No real reason to export the object mapping of original reducers in this case.